kaldb/src/test/java/com/slack/kaldb/metadata/cache/CacheSlotMetadataSerializerTest.java

```diff
@@ -33,6 +37,8 @@ public void testCacheSlotMetadataSerializer() throws InvalidProtocolBufferExcept
     assertThat(deserializedCacheSlotMetadata.cacheSlotState).isEqualTo(cacheSlotState);
     assertThat(deserializedCacheSlotMetadata.replicaId).isEqualTo(replicaId);
     assertThat(deserializedCacheSlotMetadata.updatedTimeEpochMs).isEqualTo(updatedTimeEpochMs);
+    assertThat(deserializedCacheSlotMetadata.supportedIndexTypes).containsAll(supportedIndexTypes);
```

Comment:
This could be a containsexactly instead of two asserts

Reply:
Changed.